extensions/blocks/contact-info/class-jetpack-contact-info-block.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Class Jetpack_Contact_Info_Block
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Helper class that lets us add schema attributes dynamically because they are not something that is store with the content.
+ * Due to the limitations of wp_kses.
+ *
+ * @since 7.1.0
+ */
+class Jetpack_Contact_Info_Block {
+
+	/**
+	 * Adds contact info schema attributes.
+	 *
+	 * @param array  $attr    Array containing the contact info block attributes.
+	 * @param string $content String containing the contact info block content.
+	 *
+	 * @return string
+	 */
+	public static function render( $attr, $content ) {
+		Jetpack_Gutenberg::load_assets_as_required( 'contact-info' );
+
+		return str_replace(
+			'class="wp-block-jetpack-contact-info"',
```

Comment:
This test (and similar `class` tests for other child blocks here) will break if the user defines a custom `className` attribute.

You'll either need to disable `className` support for the block or pick it from `$attr` and include as part of the `str_replace`.

<img width="281" alt="image" src="https://user-images.githubusercontent.com/87168/53256033-a112ba00-36cf-11e9-98c9-a3aaa9b3569f.png">

Reply:
Should be fixed now.